middleware/logging.go

```diff
@@ -32,7 +32,7 @@ func (l Log) Wrap(next http.Handler) http.Handler {
 		}
 		i := &interceptor{ResponseWriter: w, statusCode: http.StatusOK}
 		next.ServeHTTP(i, r)
-		if 100 <= i.statusCode && i.statusCode < 400 {
+		if 100 <= i.statusCode && (i.statusCode < 400 || i.statusCode < 429) {
```

Comment:
Maybe for that case we can use metrics to discover whether that's happening (perhaps adding an alert if we care enough), and then restarting in with debug logging when there's a problem we want to debug?